common.json

```diff
@@ -19,6 +19,20 @@
 			"files": [ "**/*.yaml", "**/*.yml" ],
 			"parser": "yaml-eslint-parser",
 			"extends": "./yaml"
+		},
+		{
+			"files": [
+				"**/*.ts"
+			],
+			"extends": [
+				"./typescript.js"
```

Comment:
The `.js` is not necessary here.

Maybe use the same formatting as the other overrides above (use one line for files and extends)

Reply:
Done